lib/DnD/helpers/getItemStyles.js

```diff
@@ -0,0 +1,18 @@
+function getItemStyles(initialOffset, currentOffset) {
+  if (!initialOffset || !currentOffset) {
+    return {
+      display: 'none'
+    };
+  }
+
+  const { x, y } = currentOffset;
+
+  const transform = `translate(${x}px, ${y}px)`;
```

Comment:
I had to tweak this a few times depending on what we were previewing, we may need to make this configurable in the future